cluster/create-subscription.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# Copyright 2021 The TestGrid Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o nounset
+set -o errexit
+
+topic=
+project=
+ack=10
+exp=24h
+canary=
+
+while getopts "t:p:a:e:c" flag; do
+    case "$flag" in
+        t) topic="$OPTARG";;
+        p) project="$OPTARG";;
+        a) ack="$OPTARG";;
+        e) exp="$OPTARG";;
+        c) canary=1;;
+    esac
+done
+
+shift $((OPTIND -1))
+
+if [[ $# -lt 1 || -z "$topic" ]]; then
+    echo "Usage: $(basename "$0") [-a $ack] [-e $exp] -c [-p $project] <-t TOPIC> SUBSCRIPTION [EXTRA ...]" >&2
+    echo >&2
+    echo "  -a ACK  : seconds to allow testgrid to ack a message" >&2
```

Comment:
Should we recommend defaults for any of these? (I think everything but -p and maybe -t could have a reasonable default)

Reply:
If you check  L22-23 and L39 you'll see that we actually do have a default and print it.